ledger-tool/src/main.rs

```diff
@@ -2107,6 +2107,36 @@ fn main() {
                                 _ => Some(value_t_or_exit!(arg_matches, "hashes_per_tick", u64)),
                             });
                         }
+
+                        for address in feature_gates_to_deactivate {
```

Comment:
@brooksprumo we only do this `for address ...` in the case of:
`if child_bank_required {`

Reply:
I think that's fine... up on line 2087 `child_bank_required` will always be true if `!feature_gates_to_deactivate.is_empty()`

Reply:
roger roger.